ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -187,3 +187,39 @@ def test_paddle_ifftshift(
         x=x[0],
         axes=axes,
     )
+
+
+# rfft
+@handle_frontend_test(
+    fn_tree="paddle.fft.rfft",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        min_dim_size=2,
+        shape=helpers.get_shape(min_num_dims=1,
+                                max_num_dims=2,
+                                min_dim_size=2,
+                                max_dim_size=4,),
```

Comment:
@asad0991 Perhaps you forgot to install/run `pre-commit` to fix your [lint/formatting errors](https://github.com/unifyai/ivy/actions/runs/5983491732/job/16233689761?pr=19454)?